apps/studio/src/components/ResourceSelector/ResourceItem.tsx

```diff
@@ -92,13 +101,13 @@ export const ResourceItem = (props: ResourceItemProps) => {
         <ErrorBoundary
           onReset={reset}
           fallbackRender={({ resetErrorBoundary }) => (
-            <Box>
+            <div>
```

Comment:
no action required, just fyi: i think chakra box is actually just `chakra(div)`, which essentialliy means that it's a div that allows us to use props shorthand. this change shouldn't have any impact (and no need to change back) but was wondering why it was made

Reply:
not much reason haha, just thought that since we aren't using the feature of `Box` then a bit pointless to import it